<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3DViewer_v1.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(80, 80, 80);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_open">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: white;
  background-color: rgb(200, 150, 50);
  border: 1px solid black;
}
QPushButton:pressed {
    background-color: rgb(150, 100, 0);
}</string>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_scale">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>120</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Scale:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_scale">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>160</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: darkgreen;
background-color: white;
border:1px solid black</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scale_minus">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>120</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
  qproperty-icon: url(://Icons/zoom-out.png)
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scale_plus">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>120</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
  qproperty-icon: url(://Icons/zoom-in.png)
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_move">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>200</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Move:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_move">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>240</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: darkgreen;
background-color: white;
border:1px solid black</string>
    </property>
    <property name="text">
     <string>0.2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_moveX_plus">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>240</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
  qproperty-icon: url(://Icons/angle-right.png)
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_moveX_minus">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>240</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
  qproperty-icon: url(://Icons/angle-left.png)
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_rotate">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>320</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Rotate:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rotateX_minus">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>400</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
  qproperty-icon: url(://Icons/angle-circle-down.png)
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_rotate">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>360</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: darkgreen;
background-color: white;
border:1px solid black</string>
    </property>
    <property name="text">
     <string>0.2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rotateX_plus">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>320</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
  qproperty-icon: url(://Icons/angle-circle-up.png)
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_moveY_minus">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>280</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
  qproperty-icon: url(://Icons/angle-down.png)
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_moveY_plus">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>200</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
  qproperty-icon: url(://Icons/angle-up.png)
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_moveZ_minus">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>280</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
  qproperty-icon: url(://Icons/angle-in.png)
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_moveZ_plus">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>200</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
  qproperty-icon: url(://Icons/angle-out.png)
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rotateY_minus">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>360</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
  qproperty-icon: url(://Icons/angle-circle-left.png)
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rotateY_plus">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>360</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
  qproperty-icon: url(://Icons/angle-circle-right.png)
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rotateZ_minus">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>400</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
  qproperty-icon: url(://Icons/angle-circle-in.png)
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rotateZ_plus">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>320</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
  qproperty-icon: url(://Icons/angle-circle-out.png)
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="ObjectViewer" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>820</width>
      <height>760</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scale">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>160</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
  qproperty-icon: url(://Icons/search.png)
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_readOnly_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  	color: white; 
	background-color: rgba(255, 255, 255, 0);
}</string>
    </property>
    <property name="text">
     <string>Название файла:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_readOnly_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  	color: white; 
	background-color: rgba(255, 255, 255, 0);
}</string>
    </property>
    <property name="text">
     <string>Количество вершин:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_readOnly_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  	color: white; 
	background-color: rgba(255, 255, 255, 0);
}</string>
    </property>
    <property name="text">
     <string>Количество граней:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fileName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>680</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  	color: white; 
	background-color: rgba(255, 255, 255, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_countV">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  	color: white; 
	background-color: rgba(255, 255, 255, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_countF">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  	color: white; 
	background-color: rgba(255, 255, 255, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_color">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>50</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_color_background">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>80</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>BG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_color_v">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>80</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_color_f">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>80</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_color_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>520</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Size of:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sizeV">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>560</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>vertex</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sizeF">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>600</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>facets</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sizeV">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>560</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: darkgreen;
background-color: white;
border:1px solid black</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sizeF">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>600</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: darkgreen;
background-color: white;
border:1px solid black</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sizeV_ok">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>560</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sizeF_ok">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>600</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}</string>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_drawV">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>450</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Vertex draw</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_drawF">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>480</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Facets draw</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_take_screenshoot">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
  qproperty-icon: url(:/Icons/camera.png)
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_screencast">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200, 200, 200);
  border: 1px solid black;
  qproperty-icon: url(:/Icons/video-camera-alt.png)
}
QPushButton:pressed {
    background-color: rgb(100, 100, 100);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_dotted">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>680</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Dotted facets</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_central_projection">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>710</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Central projection</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_circle_vertex">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>650</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Circle vertex</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ObjectViewer</class>
   <extends>QWidget</extends>
   <header location="global">objectviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
